server/lib/decoder/decoder.ex

```diff
@@ -190,6 +189,7 @@ defmodule Realtime.Decoder do
         0 -> []
         1 -> [:cascade]
         2 -> [:restart_identity]
+        3 -> [:cascade, :restart_identity]
```

Comment:
These options aren't used anywhere so we can consider removing them entirely from `%Truncate{}`. Adding this in so it doesn't raise an error when truncating with Cascade and Restart Identity options.